### ISSUE TITLE
Can the Volume Buttons be added as commands?

### ISSUE THREAD

**Issue by jat80** [lprhodes/homebridge-apple-tv#17]
Just wondering if this can be done? @lprhodes

**Comment by MrDrache333** [comment]
Its about how you control your Volume of the AppleTV...if you can control it via hddmi or WLan it could not be a problem...but if you control your Suroundsystem via IR then you logically need a transmitter on xour pi to send those codes

**Comment by normen** [comment]
Yeah, the AppleTV doesn't really have a "volume" per se, it either controls the TV via infrared through the remote or it controls the TV via HDMI. You should look into controlling your TV/amp/soundsystem, that's what I'm doing.

**Comment by epheterson** [comment]
The Apple TV does have its own volume, for like when you have headphones as your output.

**Comment by normen** [comment]
In that case it controls the (I guess bluetooth in your case?) headphone volume.

_There is no software volume in the Apple TV_ and I am very happy Apple didn't add it. Theres too many people having their gain staging completely messed up because of volume knobs and options in the silliest places.

**Comment by MrDrache333** [comment]
You can of cause log the Commands the AppleTV Remote send with IR to the Apple TV via an IR Receiver (maybe Arduino (i think i have one, than i will test that)). Than you can send the same commands via an Transceiver over your for example Raspberry Pi...than you can control the Volume

**Comment by MrDrache333** [comment]
I tested that...i wired an IR Receiver onto my Arduino Board and runned some Code ive Written and....The HEX Code, the Apple TV Remote sends are 

For Volume +:
Alternating 810 and 10 for each Press on the Button

For - Volume:
Alternating 811 and 11 for each Press on the Button

Holding the Button just Repeats the sende code

So you can of cause use this Code and Send it with an IR Diode with a Library on you Raspberry Pi

Interesting is that there are no IR Code for the other Buttons (Menu, TV, Siri and Play/Pause)....only the Volume Buttons...that underlines the Statement that the Volume COntrol is not implemented in the Remote API
